docs/proposals/ingress.md

```diff
@@ -0,0 +1,91 @@
+# Ingress Support at Edge
+
+
+## Abstract:
+
+To provide [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) support at the edge. Ingress exposes HTTP routes from outside the cluster to services within the cluster. 
+Using ingress the user can bring all their services under a single umbrella and manage them easily. The ingress support should continue to work even in offline scenarios i.e. even in the cases where 
+connectivity between edge and cloud is lost/unreliable. 
+
+
+## Motivation:
+
+The cloud native and microservice architecture is becoming more and more popular and the edge node is becoming more and more powerful. In these scenarios, users might want to deploy their applications at the edge, 
+which could need to be accessed by external users, through services. Using ingress we would be able to provide name based virtual hosting and load balance traffic.
+
+This design document deals with enabling users to leverage powerful machines at the edge to provide external users 
```

Comment:
Can you give more details of what do you mean by external user ?

Reply:
done, thanks